lib/components/mobile/results-error.js

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { Button } from 'react-bootstrap'
+import { connect } from 'react-redux'
+
+import * as formActions from '../../actions/form'
+import * as uiActions from '../../actions/ui'
+import ErrorMessage from '../form/error-message'
+
+/**
+ * This component is used on mobile views to
+ * render an error message if no results are found.
+ */
+class ResultsError extends Component {
+  static propTypes = {
+    error: PropTypes.object,
+    setMobileScreen: PropTypes.func
+  }
+
+  _editSearchClicked = () => {
+    this.props.clearActiveSearch()
+    this.props.setMobileScreen(uiActions.MobileScreens.SEARCH_FORM)
+  }
+
+  render () {
+    const { error, style } = this.props
+    return (
+      <div className='results-error-message' style={style}>
```

Comment:
If instead a `className` prop is passed, this higher-order component can be styled as a styled component. [See these docs](https://styled-components.com/docs/advanced#caveat).

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/339/commits/d7737831bffd7e6a4f5294541b0187c1a2574c3a.